observations/obs_converters/SIF/SIF_to_obs_netcdf.rst

```diff
@@ -0,0 +1,204 @@
+PROGRAM ``SIF_to_obs_netcdf``
+=============================
+
+Overview
+--------
+
+Harmonized SIF data product to DART observation sequence converter
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This routine converts a harmonized satellite SIF product 
+(Harmonized long-term SIF; SIF005) to DART ``obs_seq`` format.
+The SIF product is described by
+`JPL <https://climatesciences.jpl.nasa.gov/sif/download-data/level-3/>`__ 
+and combines GOME-2 and SCIAMACHY SIF retrievals, along with MODIS data
+to produce a single continuous, monthly, 0.05 degree SIF data set.  
+See `Wen et al., 2020 RSE <https://doi.org/10.1016/j.rse.2020.111644>`__ 
+for a more detailed description.  Download is made available by the authors
+`here <https://cornell.app.box.com/s/gkp4moy4grvqsus1q5oz7u5lc30i7o41/folder/100438579357>`__.
+The conversion script was designed and tested for version SIF005v2. 
+
+This SIF data product also comes with its own uncertainty value, and quality 
+control flag described below.  Namelist options also include a wavelength option
+(e.g. 740 nm or 755 nm) to specify the location the SIF value is centered upon. 
+
+
+Standard workflow:
+
+#. Download the Level 3 data for the months of interest. Years 2002-2018 are available
+   as of 5/18/21.  (see 'Data Source' below)
+#. Make note of the SIF wavelength the data is centered upon. This information is 
+   included in the SIF variable of netcdf file ``SIF_740_daily_corr``  
+#. Build the DART executables with support for land observations. This is done by running 
+   ``preprocess`` with ``obs_def_lnd_mod.f90`` in the list of ``input_files`` for 
+   ``preprocess_nml``.
+#. Provide basic information via the ``SIF_to_obs_netcdf_nml`` (e.g. verbose, wavelength)
+#. Convert single or multiple SIF netcdf data files using ``SIF_to_obs_netcdf``
+#. Combine all output files for the region and timeframe of interest into one file using
+   :doc:`../../../assimilation_code/programs/obs_sequence_tool/obs_sequence_tool`
+
+For some models (CLM, for example), it is required to reorganize the observation sequence 
+files into a series of files that contains ONLY the observations for each assimilation. 
+This can be achieved with the `makedaily.sh <makedaily.sh>`__ script.
```

Comment:
```suggestion
This can be achieved with the `makedaily.sh` script which can be found in the `DART/models/clm/shell_scripts` directory
```

The makedaily link does not go anywhere, so suggesting a note to look in the clm/shell_scripts directory